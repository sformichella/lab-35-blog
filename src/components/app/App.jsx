import React from 'react';

import { PostProvider } from '../../state/PostProvider';
import PostForm from '../PostForm/PostForm';
import PostList from '../PostList/PostList';

export default function App() {
  return (
    <PostProvider>
      <PostForm />
      <PostList />
    </PostProvider>
  );
}
