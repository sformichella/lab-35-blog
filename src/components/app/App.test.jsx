import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';
import { Provider } from 'react-redux';
import store from '../../store';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <App />
      </Provider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
