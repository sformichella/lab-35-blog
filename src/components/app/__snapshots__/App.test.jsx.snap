// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <form>
    <label
      for="title"
    >
      Title:
    </label>
    <input
      id="title"
      required=""
      type="text"
      value=""
    />
    <label
      for="body"
    >
      Body:
    </label>
    <input
      id="body"
      required=""
      type="text"
      value=""
    />
    <button>
      Post!
    </button>
  </form>
  <ul />
</DocumentFragment>
`;
